content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -454,15 +454,15 @@ model Model {
 
 #### CockroachDB (Preview)
 
-| Native database type | Native database type attribute | Notes |
-| :------------------- | :----------------------------- | ----- |
-| `STRING` \| `TEXT`   | `@db.Text`                     |       |
-| `CHAR`               | `@db.Char(x)`                  |       |
-| `VARCHAR`            | `@db.VarChar(n)`               |       |
-| `BIT`                | `@db.Bit(x)`                   |       |
-| `VARBIT`             | `@db.VarBit`                   |       |
-| `UUID`               | `@db.Uuid`                     |       |
-| `INET`               | `@db.Inet`                     |       |
+| Native database type              | Native database type attribute | Notes |
+| :-------------------------------- | :----------------------------- | ----- |
+| `STRING` \| `TEXT` \| `VARCHAR`   | `@db.String`                   |       |
+| `CHAR`                            | `@db.Char(x)`                  |       |
+| `"char"`                          | `@db.SingleChar`               |       |
```

Comment:
is the lowercase spelling and explicit double quotes around it intentional?

Reply:
yes, it's a very weird type

Reply:
_very_ weird indeed.

Reply:
it's documented somewhere in https://www.cockroachlabs.com/docs/v21.2/string